e2e/next-sandbox/test/comments/with-suspense.test.ts

```diff
@@ -22,21 +22,25 @@ test.describe("Comments", () => {
   test.afterEach(() => Promise.all(pages.map((page) => page.close())));
 
   test("verify A and B display same number of threads after threads are loaded", async () => {
-    await waitForJson(pages, "#isLoading", false);
-    await waitUntilEqualOnAllPages(pages, "#numOfThreads");
+    await waitForJson(pages, "#isLoading", false, { timeout: 15_000 });
+    await waitUntilEqualOnAllPages(pages, "#numOfThreads", { interval: 250 });
   });
 
   test("verify thread creation on B is broadcasted correctly to A", async () => {
     const [page1, page2] = pages;
 
-    await waitForJson(pages, "#isLoading", false);
+    await waitForJson(pages, "#isLoading", false, { timeout: 15_000 });
+    await waitUntilEqualOnAllPages(pages, "#numOfThreads", { interval: 250 });
 
-    const numOfThreads = (await getJson(page1, "#numOfThreads")) as number;
+    // Read starting value n
+    const n = (await getJson(page1, "#numOfThreads")) as number;
 
+    await page1.click("#create-thread");
     await page2.click("#create-thread");
-    await waitForJson(page1, "#numOfThreads", numOfThreads + 1);
+    await page2.click("#create-thread");
+    await waitForJson(pages, "#numOfThreads", n + 3, { timeout: 15_000 });
 
     await page2.click("#delete-comment");
-    await waitUntilEqualOnAllPages(pages, "#numOfThreads");
+    await waitForJson(pages, "#numOfThreads", n + 3 - 1, { timeout: 15_000 });
```

Comment:
Hmm. Actually, even with a 15 second timeout, this line keeps failling. Maybe there is an actual issue here after all? It looks like it's pretty common that this comment deletion would not reach the other end of the line!

Reply:
I'd encourage you to download the Playwright report ZIP file from CI and open it locally to inspect the full trace there. You can see all the details, including recorded network traffic, etc. Let me know if you need help doing that, happy to help.

Reply:
The tests seem to pass most of the times on my machine locally. But, the tests failing/passing is inconsistent on GitHub. Can't seem to understand properly why this may be happening

Reply:
One way I found pretty effective to capture this and get a good trace of it locally is to run the following:

```fish
while env HEADLESS=1 turbo run test -- comments; end
```

This will run the test suite in a loop until it fails. This one just took me 3 runs to get:

![Screen Shot 2023-10-20 at 08 38 31@2x](https://github.com/liveblocks/liveblocks/assets/83844/b232d566-c677-46af-a0b7-a3dff83b6083)

Maybe there is an actual bug here that it's trying to tell us about after all, instead of just infra slowness?